rakelib/release.rake

```diff
@@ -0,0 +1,158 @@
+# encoding: utf-8
+require "fileutils"
+
+LOGSTASH_BUILD_PATH = File.expand_path(File.join(File.dirname(__FILE__), "..", "build"))
+
+LOGSTASH_CORE_PLUGIN_API_DEFAULT_VERSION = "1.0"
```

Comment:
can you place this in the plugin api gem code itself? maybe under lib/logstash-plugin-api/version.rb ?